source/documentation/doc-Introduction_to_the_VM_Portal/topics/Connecting_to_virtual_machines.adoc

```diff
@@ -11,3 +11,8 @@
 . Open the file with *remote-viewer*. A console window displays.
 +
 You can now use the virtual machine in the same way you would use a physical desktop.
+
+[NOTE]
+====
+When you are finished using the console, you must restart the Virtual Machine. Otherwise, the console session continues and is unavailable to other users.
```

Comment:
this note seems to cover a very specific use case where the vm holds user data (therefore `Disable strict user checking` is not set).
maybe a better phrasing would be something like: in order to allow other users to connect to the VM, when you are finished using the console make sure to restart the virtual machine